src/rules/noEmptyRule.ts

```diff
@@ -68,4 +67,10 @@ class BlockWalker extends Lint.RuleWalker {
 
         super.visitConstructorDeclaration(node);
     }
+
+    private isPropertyAccessModifier(modifier: string): boolean {
+        return modifier === ts.SyntaxKind.PrivateKeyword ||
+            modifier === ts.SyntaxKind.ProtectedKeyword ||
+            modifier === ts.SyntaxKind.PublicKeyword;
```

Comment:
can you please put the `||` operator on the next lines?

(sorry, we should probably have a tslint rule for this)

```ts
return modifier === ts.SyntaxKind.PrivateKeyword
    || modifier === ...
```